README.rst

```diff
@@ -2,3 +2,117 @@ cdist manifestation
 ===================
 
 A helper for writing cdist manifests in Python.
+
+Motivation
+----------
+
+Writing complex manifests in shell can be painful and laborious. With Python and
+a proper abstraction layer, we can write manifests more easily.
+
+Usage
+-----
+
+First of all make sure your manifest is executable and contains
+``#!/usr/bin/env python`` shebang.
+
+Types
```

Comment:
This section is missing mention about implicit underscore to dash conversion in parameters.

Reply:
This is mentioned at the bottom of this section now.